ArmPkg/Drivers/MmCommunicationDxe/MmCommunication.c

```diff
@@ -14,6 +14,7 @@
 #include <Library/HobLib.h>
 #include <Library/PcdLib.h>
 #include <Library/UefiBootServicesTableLib.h>
+#include <Library/UefiLib.h>

```

Comment:
Comment on commit message:
"that Mm requires"?
Then why wasn't it there in the first place?
Is this a new requirement?

Reply:
@leiflindholm 
_Then why wasn't it there in the first place?_ 
I do not know
_Is this a new requirement?_
It has been defined in PI SPEC for years. The driver has events on EfiEndOfDxeEvent, EfiEventExitBootServices, and EfiEventReadyToBoot, but it is missing protocol notification on gEfiDxeMmReadyToLockProtocolGuid, which should trigger MMI handle to install gEfiMmReadyToLockProtocolGuid protocol in Mm.